base-images-sbom-script/README.md

```diff
@@ -0,0 +1,60 @@
+# base images sbom script
+
+This is a script that creates sbom data for base images. It is used in 
+[buildah task](https://github.com/redhat-appstudio/build-definitions/tree/main/task/buildah) in Konflux pipelines.
+
+It takes several inputs:
+1. path to the sbom file, that will be updated in place with the base image data
+2. path to a file containing base images as taken from from the dockerfile (with preserved order)
+3. path to a file containing base images with digests, generated from the output of **buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}'**. The dockerfile order must be preserved as well
+
+
+
+The base images data will be added in the [formulation attribute](https://cyclonedx.org/docs/1.5/json/#formulation), by appending new item with the **components** array.
+This will not affect any other items in the formulation array. If the formulation part of the sbom does not exist, it is created.
+The result might look like this:
+
+```
+    ...
+    "formulation": [
+        {
+            "components": [
+                {
+                    "type": "container",
+                    "name": "quay.io/mkosiarc_rhtap/single-container-app",
+                    "purl": "pkg:oci/single-container-app@sha256:8f99627e843e931846855c5d899901bf093f5093e613a92745696a26b5420941?repository_url=quay.io/mkosiarc_rhtap/single-container-app&tag=f2566ab",
+                    "properties": [
+                        {
+                            "name": "builder-0",
+                            "value": 0
```

Comment:
The property name needs an update to match the code

Reply:
updated